pkg/deps/config/suppliers.go

```diff
@@ -286,10 +298,7 @@ func NewSupplyRingCacheFn() SupplierFn {
 // NewSupplyPOKTRollSDKFn returns a function which constructs a
 // POKTRollSDK instance with the required dependencies and returns a new
 // depinject.Config which is supplied with the given deps and the new POKTRollSDK.
-func NewSupplyPOKTRollSDKFn(
-	queryNodeURL *url.URL,
```

Comment:
Were we just not using this?

Reply:
Yes, it as an artifact. In the `AppGateServer` case we get these from depinject. They are needed as config entries in the library mode though where the SDK self supplies its deps.